### ISSUE TITLE
图片上有红色波浪线

### ISSUE THREAD

**Issue by spawpaw** [apache/dubbo-website#441]
![图片](https://user-images.githubusercontent.com/18007922/62434677-d7b72b00-b76b-11e9-916d-12ecc669cf47.png)
[config-center.html](http://dubbo.apache.org/zh-cn/docs/user/configuration/config-center.html)
不止这一个页面，好多图都有

**Comment by caojiele** [comment]
> ![图片](https://user-images.githubusercontent.com/18007922/62434677-d7b72b00-b76b-11e9-916d-12ecc669cf47.png)
> [config-center.html](http://dubbo.apache.org/zh-cn/docs/user/configuration/config-center.html)
> 不止这一个页面，好多图都有

Hi, It looks like that editor opened the spelling check, can you solve it?

**Comment by spawpaw** [comment]
of course, but I don't have the image's original file, where can I get it?

**Comment by caojiele** [comment]
> of course, but I don't have the image's original file, where can I get it?

Article has corresponding image links, I suggest you have a problem first article list, and then modified.

**Comment by caojiele** [comment]
> of course, but I don't have the image's original file, where can I get it?

Suggest you can ask the author of this article.
